addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -7,17 +7,24 @@ var widgetsMedia = require('wysiwyg.widgets.media');
 
 var unsplashAPI = null;
 
+// Prevent base class from treating unsplash images like regular attachments
+const originalEvents = widgetsMedia.ImageWidget.prototype.events;
+const clickHandler = originalEvents['click .o_existing_attachment_cell'];
+_.extend(originalEvents, {
+    'click .o_existing_attachment_cell:not(.o_unsplash_attachment_cell)': clickHandler,
+});
+delete originalEvents['click .o_existing_attachment_cell'];
```

Comment:
Maybe we can make this more robust: I feel like if someone wants to change the original selector into `'click .o_existing_attachment_cell, .hello'` for whatever reason, it should not have to grep to see if the exact string was used somewhere else to be changed.

Either:
- Change here to throw an error if `clickHandler` is not defined
- Change the original selector to already contain a `:not` with a generic class you use in unsplash

Reply:
What do you think about defining the attachment cell selector as a static field on the original widget? I can override it here, and if someone changes it there, there's no need to come change it here as I'd just tack on a `:not(.o_unsplash_attachment_cell)` at the end.

Reply:
Works too but less robust. If someone changes `'click A'` into `'click A, B'` (or even a working typo like `'click C '`), your `:not` won't work as intended. However, still more robust than what we have now since if someone changes the content of a variable, he should grep to see where and how it used. So, as you wish.

Reply:
Alright, went with throwing an error. I don't like breaking the encapsulation of the original module to accommodate for the unsplash extension.